posthog-android/src/main/java/com/posthog/android/replay/PostHogReplayIntegration.kt

```diff
@@ -556,14 +572,11 @@ public class PostHogReplayIntegration(
             if (!view.isNoCapture(config.sessionReplayConfig.maskAllImages)) {
                 // TODO: we can probably do a LRU caching here for already captured images
                 view.drawable?.let { drawable ->
-                    // IconicsDrawable always return null to copy so we fallback to the original one
-                    val drawableCopy = drawable.copy(view.resources) ?: drawable
-                    val convertedBitmap = config.sessionReplayConfig.drawableConverter?.convert(drawableCopy)
-                    base64 = if (convertedBitmap != null) {
-                        convertedBitmap.base64()
-                    } else {
-                        drawableCopy.base64(view.width, view.height)
-                    }
+                    base64 = drawable.cloneAndToBase64(view)
+//                    style.paddingTop = view.paddingTop.densityValue(displayMetrics.density)
+//                    style.paddingBottom = view.paddingBottom.densityValue(displayMetrics.density)
+//                    style.paddingLeft = view.paddingLeft.densityValue(displayMetrics.density)
+//                    style.paddingRight = view.paddingRight.densityValue(displayMetrics.density)
```

Comment:
need to keep the commented code?

Reply:
yes, but it is a TODO (have to figure out the offset), the padding fixes some cases and breaks others because of the drawable size (vectors), the OS adjusts to the right size but we don't have access to this info, so in some cases the padding will fix it, in some cases it will break it.